### ISSUE TITLE
Contains lookup doesn't seem to support LIKE 'value%' SQL [was ORM Equivalent of SQL]

### ISSUE THREAD

**Issue by DavidMuller** [djangonauts/django-hstore#75]
Hello, I'm having trouble forming a django ORM query for a query.  In plain english, the query sounds like: "select a row where data[my_key] contains the string I'm looking for."

Concretely, consider the following model:

``` python
from django.db import models
from django_hstore import hstore


class TestHstore(models.Model):
    name = models.CharField(max_length=250)
    data = hstore.DictionaryField(db_index=True)
    objects = hstore.HStoreManager()

    def __unicode__(self):
        return self.name
```

The following model instance:

``` python
instance = TestHstore.objects.create(name="test", data={"en_us": "english", "french": "fries"})
```

I'd like to make use the Django ORM that makes a query equivalent to the following SQL: 

```
SELECT * 
FROM hstore_test_testhstore 
WHERE data->'french' LIKE 'fr%';
```

I can't seem to find a way to do that within the ORM.  `TestHstore.objects.filter(data__contains={'en_us': "english"})` works for exact matches of the value in the key/value pair, but fails to return a database row when the value doesn't match completely (`TestHstore.objects.filter(data__contains={'en_us': "eng"})`)

Any  Suggestions?

Thanks for this cool package--really great work!

**Comment by nemesisdesign** [comment]
Sorry for the delay in the answer, we had many issue to resolve. At the moment this is not possible so I marked this as a feature request.
